gateway/src/test/java/io/camunda/zeebe/gateway/api/job/LongPollingActivateJobsTest.java

```diff
@@ -510,6 +521,52 @@ public void shouldReturnJobsIfSomeBrokersHaveJobsWhileOthersReturnResourceExhaus
     assertThat(response.getJobsList()).hasSize(10);
   }
 
+  @Test
+  public void shouldRepeatRequestOnlyOnce() throws Exception {
+    // given
+    // the first three requests activates jobs
+    final var firstRequest = getLongPollingActivateJobsRequest();
+    final var secondRequest = getLongPollingActivateJobsRequest();
+    final var thirdRequest = getLongPollingActivateJobsRequest();
+    // the last request does not activate any jobs
+    final var fourthRequest = getLongPollingActivateJobsRequest();
+
+    final var allRequestsSubmittedLatch = new CountDownLatch(1);
```

Comment:
Dear Reviewer, just FYI: I consciously decided to use the `CountDownLatch` to ensure that all requests are submitted to the actor initially. As an alternative, I considered using the `ControlledActorScheduler` but the test was too flaky with the controlled scheduler. With the latch, the test is reliable.